i18n/hi.toml

```
@@ -20,21 +20,21 @@ other = "में"
 
 # Phrases for tags
 [ui_see_all_tags]
-other = "See all tags"
+other = "देखना सभी टैग"
 [ui_tag]
-other = "Tag"
+other ="टैग"
```

We don't need to remove the spaces. Let's preserve.

```suggestion
other = "टैग"
```